Project.toml

```diff
@@ -3,6 +3,12 @@ uuid = "b3c78db8-5351-4c94-92cb-e6a7a02f3abe"
 authors = ["JieLi"]
 version = "0.1.0"
 
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+NiLang = "ab4ef3a6-0b42-11ea-31f6-e34652774712"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
```

Comment:
Random is only used in testing. You should put it in the `extra` field. e.g. 
https://github.com/QuantumBFS/YaoArrayRegister.jl/blob/9dd04982359435bca2f93679c23824420229799b/Project.toml#L24

Reply:
Thanks and I have corrected it in my local workspace~